spec/services/twilio_service_spec.rb

```diff
@@ -50,28 +50,28 @@
       TwilioService.new
     end
 
-    it 'sends an OTP from the test number when pt_mode is true' do
+    it 'sends an OTP from the test number when pt_mode is true', sms: true do
       expect(FeatureManagement).to receive(:pt_mode?).at_least(:once).and_return(true)
       SmsSenderOtpJob.perform_now(user)
 
       expect(messages.first.from).to eq '+15005550006'
     end
 
-    it 'sends an OTP from the real number when pt_mode is false' do
+    it 'sends an OTP from the real number when pt_mode is false', sms: true do
       expect(FeatureManagement).to receive(:pt_mode?).at_least(:once).and_return(false)
       SmsSenderOtpJob.perform_now(user)
 
       expect(messages.first.from).to eq '+19999999999'
```

Comment:
Good point, but I think that's beyond the scope of this PR. The twilio keys in application.yml are not used in practice because we have the correct keys in secrets.yml, but we have the `self.singular_account` method in `TwilioService` that falls back to the keys in application.yml if the ones in secrets.yml are not present. So, they are actually used in a sense.

We can open an issue to address this later. Ideally, we would do something like the required keys initializer that Figaro provides so that the app won't run if keys are missing.